go/storage/client/client.go

```diff
@@ -48,6 +64,8 @@ type storageClientBackend struct {
 
 	nodesClient grpc.NodesClient
 	runtime     registry.RuntimeDescriptorProvider
+
+	backendOverrides map[signature.PublicKey]api.Backend
```

Comment:
Maybe add a note that this doesn't need to be behind a lock as long as it's only updated via options on initialization.

Reply:
Done.